google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -566,6 +566,48 @@
     <method>java.util.List getFloat32Array()</method>
   </difference>
 
+  <!-- UUID -->
+  <difference>
+    <differenceType>7013</differenceType>
+    <className>com/google/cloud/spanner/AbstractStructReader</className>
+    <method>java.util.UUID getUuidInternal(int)</method>
+  </difference>
+  <difference>
+    <differenceType>7013</differenceType>
+    <className>com/google/cloud/spanner/AbstractStructReader</className>
+    <method>java.util.List getUuidListInternal(int)</method>
+  </difference>
```

Comment:
I don't see AbstractStructReader methods added for other types in CLIRR file. Why was it needed for this type?
Can you remove these and check if CLIRR succeeds?

Reply:
When removed, it doesn't succeed.

Reply:
Got it, looks like to preserve compatibility, we will have to provide a default implementation in AbstractStructReader instead of making it abstract.
Can you change it similar to the default implementation of getPgJsonbInternal() method in the same class?
Something like
```
  protected UUID getUuidInternal(int columnIndex) {
    throw new UnsupportedOperationException("Not implemented");
  }
```

Reply:
Done.